Makefile.inc

```diff
@@ -0,0 +1,39 @@
+GO ?= go
+EPOCH_TEST_COMMIT ?= 1cc5a27
+PROJECT := github.com/kubernetes-incubator/cri-o
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
+GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
+CRIO_IMAGE := crio_dev$(if $(GIT_BRANCH_CLEAN),:$(GIT_BRANCH_CLEAN))
+CRIO_INSTANCE := crio_dev
+PREFIX ?= ${DESTDIR}/usr/local
+BINDIR ?= ${PREFIX}/bin
+LIBEXECDIR ?= ${PREFIX}/libexec
+MANDIR ?= ${PREFIX}/share/man
+ETCDIR ?= ${DESTDIR}/etc
+ETCDIR_CRIO ?= ${ETCDIR}/crio
+BUILDTAGS ?= seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_installed.sh) $(shell hack/libdm_no_deferred_remove_tag.sh) $(shell hack/btrfs_installed_tag.sh) $(shell hack/ostree_tag.sh) $(shell hack/selinux_tag.sh)
```

Comment:
Can this (and the other Go-specific stuff) either stay in the root `Makefile` or move into a Go-specific include?  Otherwise it slows down the `conmon` `Makefile` for no purpose.